Elements/src/Spatial/AdaptiveGrid/AdaptiveGraphRoutingWeightModifier.cs

```diff
@@ -51,14 +55,43 @@ public void ClearWeightModifiers()
             _weightModifiers.Clear();
         }
 
+        /// <summary>
+        /// Set factor aggregator function for weight modifiers group.
```

Comment:
let's describe a little bit more about what these do.  "The aggregator function will be applied to any list of weight modifications that share a group name.  This ensures that if a type of modifier is applied to a single edge more than once we have options for how to apply duplicate modifiers."